src/viam/sdk/module/service.cpp

```diff
@@ -208,7 +214,9 @@ Dependencies ModuleService::get_dependencies_(
 
 std::shared_ptr<Resource> ModuleService::get_parent_resource_(const Name& name) {
     if (!parent_) {
+        // LS: I think maybe this is never hit
```

Comment:
I think you're right, I had the same thought working on this a while back. But it's harmless and this comment highlights it so probably fine to leave as-is!